custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -17,9 +17,9 @@
 TEST_CASE_TYPE = 'test'
 
 
-def validate_number(string_value):
+def validate_phone_number(string_value):
     if string_value is None or string_value.strip() == '':
-        return None
+        return ''
     else:
         return int(string_value)
```

Comment:
@proteusvacuum ok, one option i see is sometimes people will choose to store the original/locally formatted number as one case property (since sometimes people prefer to see that in commcare if it comes up), and then store the internationally formatted number as contact_phone_number for messaging